lib/SGN/Controller/AJAX/Pedigrees.pm

```diff
@@ -280,11 +280,12 @@ sub get_full_pedigree_GET {
     my $schema = $c->dbic_schema("Bio::Chado::Schema");
     my $mother_cvterm = $schema->resultset("Cv::Cvterm")->find({name  => "female_parent"})->cvterm_id();
     my $father_cvterm = $schema->resultset("Cv::Cvterm")->find({name  => "male_parent"})->cvterm_id();
+    my $accession_cvterm = $schema->resultset("Cv::Cvterm")->find({name  => "accession"})->cvterm_id();
```

Comment:
generally we use $accession_cvterm_id = SGN::Model::Cvterm->get_cvterm_row($schema, 'accession', 'stock_property')->cvterm_id(); to fetch cvterms that way you also specify the cv of the cvterm

Reply:
'accession' is a 'stock_type', not a 'stock_property'